src/sagemaker/image_uri_config/data-wrangler.json

```diff
@@ -0,0 +1,33 @@
+{
+  "processing": {
+    "versions": {
+      "1.x": {
```

Comment:
Are the 1.x images guaranteed to be backward compatible?

Reply:
Correct.